pkg/daemon/daemon.go

```diff
@@ -127,6 +143,21 @@ func (dn *Daemon) Run(stopCh <-chan struct{}, exitCh <-chan error) error {
 	}
 }
 
+func (dn *Daemon) operatorConfigAddHandler(obj interface{}) {
+	dn.operatorConfigChangeHandler(&sriovnetworkv1.SriovOperatorConfig{}, obj)
+}
+
+func (dn *Daemon) operatorConfigChangeHandler(old, new interface{}) {
+	oldCfg := old.(*sriovnetworkv1.SriovOperatorConfig)
+	newCfg := new.(*sriovnetworkv1.SriovOperatorConfig)
+
+	if newCfg.Spec.LogLevel != oldCfg.Spec.LogLevel {
+		glog.Infof("Set log verbose level to: %d", newCfg.Spec.LogLevel)
```

Comment:
Wouldn't it be better to set a global variable (or a field of `Daemon`) to hold the verb level?
Also, I think that this is going to be written / read by different goroutines, as the two informers are spawned separately.

Reply:
There is a global variable in glog to hold the verb level, I think we can just use it. Also, the set flag call in glog it is thread-safe.